tianshou/data/batch.py

```diff
@@ -154,9 +155,8 @@ def _parse_value(v: Any):
 
 
 class Batch:
-    """Tianshou provides :class:`~tianshou.data.Batch` as the internal data
-    structure to pass any kind of data to other methods, for example, a
-    collector gives a :class:`~tianshou.data.Batch` to policy for learning.
+    """Batch the internal data structure to pass any kind of data to other \
+    methods, for example, a collector gives a batch data to policy.forward().
```

Comment:
Precise that it is a kind of supercharged array (of temporal data) stored individually in a (recursive) dictionary of object that can be either numpy array, torch tensor, or batch themself. 
It is designed to make it extremely easily to access, manipulate and set partial view of the heterogeneous data conveniently.